car_time/time.go

```diff
@@ -57,12 +57,16 @@ func (c *Car) CanShow() bool {
 func (c *Car) Render(out chan<- string) {
 	defer close(out)
 	carPaint := ansi.ColorFunc(c.GetPaint())
-
 	n := time.Now()
+
+	t := n.Format("15:04:05")
+	if h := os.Getenv("BULLETTRAIN_CAR_TIME_12HR"); h == "true" {
+		t = n.Format("3:04:05")
+	}
+
 	out <- fmt.Sprintf("%s%s",
 		paintedSymbol(),
-		carPaint(fmt.Sprintf("%02d:%02d:%02d",
-			n.Hour(), n.Minute(), n.Second())))
+		carPaint(fmt.Sprintf("%s", t)))
```

Comment:
Sprint is useless now, paint only `t` itself. `carPaint(t)`

Reply:
Sure I'll do this later today, good catch.